torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -38,9 +38,7 @@
 namespace torch {
 namespace jit {
 
-IValue to_tuple(std::vector<IValue> ivalues) {
-  return c10::ivalue::Tuple::create(std::move(ivalues));
-}
+char const* toString(OpCode op);
```

Comment:
@zhxchen17 I remember you also made this change before? Has it been landed?

Reply:
@tugsbayasgalan yeah I did, but then I realized that's not necessary for my diff so I didn't land that in the final version.